t/basic.t

```diff
@@ -56,6 +61,9 @@ done_testing();
 
```

Comment:
This is to make it stop deleting .gitignore. >:(

Reply:
FYI, you could add `t/run/*` to the root .gitignore, and get rid of t/run/.gitignore completely.